torch/jit/_trace.py

```diff
@@ -604,6 +610,7 @@ def trace(
     _force_outplace=False,
     _module_class=None,
     _compilation_unit=_python_cu,
+    example_kwarg_inputs=None
```

Comment:
my understanding is that torch.jit.trace() will _only_ work with example_kwarg_inputs if the traced object is a module (and not if it's a simple function). Is that true?

If so, can we only introduce this interface to `trace_module` for now? If you want to add support for all objects (including `torch.jit.trace(fn, example_kwarg_inputs=...)` for functions) you can do so in a separate PR if that's easier. But for now I think it would make more sense to only expose this in torch.jit.trace_module, if it only works for modules and not for functions.

Reply:
Hi, @davidberard98 
Thanks for you remainder! I did just support **module** trace in torch,jit.trace(). But I think for user it is more convenient to use torch.jit.trace() to trace a module(at least for me, I don't have to wrap the example input again(such as ``{'forward': example_inputs}``)). So I decide to complete the  
support for function in torch.jit.trace() and add the doc for ``trace_module()``.